Clustering/Hdbscan/src/test/java/org/tribuo/clustering/hdbscan/TestHdbscan.java

```diff
@@ -216,6 +240,52 @@ public void testBasicTrainPredict() {
         runBasicTrainPredict(t);
     }
 
+    @Test
+    public void deserializeHdbscanModelV42Test() {
```

Comment:
This test verifies that the predictions are the same as they are in 4.2, and using the debugger I could see that `HdbscanModel`s new `noisePointsOutlierScore` variable is set to 0.0 upon deserialization. Once the cluster exemplars are made public, we could maybe also add some quick assertions that shows calling the `getMaxDistToEdge` method returns `Double.NEGATIVE_INFINITY` to be extra careful here.

Reply:
Sounds good.